html/rendering/non-replaced-elements/the-hr-element-0/hr.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<title>The hr element</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+#ref {
+  display: block;
+  unicode-bidi: isolate;
```

Comment:
This is from Edge? Is it important? I suppose it could make a difference if the HR is inline-level, but ...